ARTwarp_Create_Figure.m

```diff
@@ -37,12 +38,22 @@
     'Label','Plot Categorisation', ...
     'Enable','off', ...
     'Tag','Plotmenu');
-h2 = uimenu('Parent',h1, ...
-    'Callback','ARTwarp_Plot_Net2', ...
-    'Accelerator','p', ...
-    'Label','Plot Categorisation 2', ...
-    'Enable','off', ...
-    'Tag','Plot2menu');
+%draw the contour and text in the top row of the figure
```

Comment:
What happened to h2

Reply:
In our program there are two ways of plotting the data, I would assume that Wendi's program only has one, hence the fact that the h2 with the label 'Plot Categorisation 2' is gone.